playbooks/setup.yml

```diff
@@ -6,6 +6,10 @@
   gather_facts: '{{ gather_facts | default("yes") }}'
   pre_tasks:
     - import_tasks: 'roles/prereqs/tasks/check_os.yml'
+    - name: "Register current git http.sslVerify value if exists"
```

Comment:
I think we need to drop the '-' here for name

Reply:
That is a separate task from the one on the line 8, hence the '-'.

Reply:
I see, but this still looks wrong imo - we cannot import_task withing pre_task. So what we could do here is add another "task" block and put the import_tasks in it after pre before post

Reply:
I'm not sure I follow - `pre_tasks` is simply a list of tasks that get executed before the roles as can be seen in [docs](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#using-roles). The task `import_tasks` is a valid task, and that line has been there since last September with no problems. My change just adds another task to the list of `pre_tasks` to be executed.

Reply:
Ok, if it worked locally we can merge. I am looking at the newer docs now, the new changes lgtm.

Reply:
Thanks!